pom.xml

```diff
@@ -81,7 +81,7 @@
         <dependency>
             <groupId>com.datastax.cassandra</groupId>
             <artifactId>cassandra-driver-core</artifactId>
-            <version>2.1.9</version>
+            <version>2.2.0-rc3</version>
```

Comment:
I believe that 2.1.9 lacks the codec registry features, which would be a pretty significant change for us. By comparison, porting this to 2.2.0-rc3 was fairly easy